cmd/miniooni/main.go

```diff
@@ -303,4 +309,11 @@ func main() {
 			}
 		}
 	}
+
+	if wrongCounter > 0 {
+		log.Infof("...Finished with %d errors!", wrongCounter)
```

Comment:
This information is computed incorrectly, as I mentioned above. Also, I believe it would be more useful to perhaps say the number of failures over the number of inputs. Also, if you structure the code in this case, you don't need to write an `if ... else` construct. You can print stats at the end, no matter what. For example:

```
Running experiment with 100 inputs. Seen 20 failures.
```